scripts/admin/decrypt-log/run.ts

```diff
@@ -0,0 +1,10 @@
+import { LogDecryption } from "../../../packages/neuron-wallet/src/services/log-encryption";
+
+const ADMIN_PRIVATE_KEY = process.env
+  .ADMIN_PRIVATE_KEY!.split(/\r?\n/)
+  .map((line) => line.trim())
+  .join("\n");
+const LOG_MESSAGE = process.env.LOG_MESSAGE!;
+
+const decryption = new LogDecryption(ADMIN_PRIVATE_KEY);
+console.log(decryption.decrypt(LOG_MESSAGE));
```

Comment:
What if using `gpg` to decrypt the `aes key` so the private key can be guaranteed by `gpg`

Reply:
If the key is protected by PGP, an admin can use its GPG client to decrypt the message instead of using this script. The script here is an option.

Reply:
Got it, please add an annotation for it because the private key is not supposed to be used directly.

Reply:
I've included a new section to provide instruction on using a protected key